scripts/make-g-cloud-7-live.py

```diff
@@ -0,0 +1,148 @@
+"""
+
+Usage:
+    scripts/make-g-cloud-7-live.py <data_api_url> <data_api_token> <draft_bucket> <documents_bucket> [--dry-run]
```

Comment:
Could we use `stage` here instead of the API URL, similar to https://github.com/alphagov/digitalmarketplace-scripts/blob/master/scripts/index-services.py#L117?